adminer/static/editing.js

```diff
@@ -271,6 +271,25 @@ function editingRemoveRow(button, name) {
 	return true;
 }
 
+/** Move table row for field
+* @param HTMLInputElement
+* @param boolean Direction to move row. True for up false for down
+* @return boolean Success
+*/
+function editingMoveRow(button, dir){
+	var row = parentTag(button, 'tr');
+
```

Comment:
> Detect

Detect `nextElementSibling` or detect `parentTag`? I believe `parentTag` is used without checks all over the place